net-misc/streamlink/streamlink-0.3.2.ebuild

```diff
@@ -1,29 +1,35 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_REQ_USE='xml(+),threads(+)'
 
 inherit distutils-r1
 
 DESCRIPTION="CLI for extracting streams from websites to a video player of your choice"
 HOMEPAGE="https://streamlink.github.io/"
 SRC_URI="https://github.com/${PN}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
-KEYWORDS="~amd64 ~arm ~x86"
-LICENSE="BSD-2"
+KEYWORDS="~amd64"
+LICENSE="BSD-2 MIT"
 SLOT="0"
 IUSE="doc test"
 
-RDEPEND="dev-python/pycrypto[${PYTHON_USEDEP}]
+RDEPEND="dev-python/pycryptodome[${PYTHON_USEDEP}]
 	dev-python/requests[${PYTHON_USEDEP}]
 	virtual/python-futures[${PYTHON_USEDEP}]
 	virtual/python-singledispatch[${PYTHON_USEDEP}]
+	dev-python/backports-shutil_which[$(python_gen_usedep 'python2*')]
+	dev-python/backports-shutil_get_terminal_size[$(python_gen_usedep 'python2*')]
+	dev-python/pycountry[${PYTHON_USEDEP}]
```

Comment:
Not enough. You need to prevent those iso* packages from creeping into egg like this:
```
python_prepare_all() {
	# Avoid iso-639, iso3166 dependencies since we use pycountry.
	export STREAMLINK_USE_PYCOUNTRY=1

	distutils-r1_python_prepare_all
}
```